assets/dev/js/editor/elements/views/behaviors/context-menu.js

```diff
@@ -73,7 +77,7 @@ module.exports = Marionette.Behavior.extend( {
 		}
 
```

Comment:
if ( 'preview' === context ) {
if ( ! isAddSectionView && ( ! this.view.container || ! this.view.container.isDesignable() ) ) {
}